### ISSUE TITLE
Pass boolean variables to Accumulo MapReduce builder methods

### ISSUE THREAD

**Issue by mikewalch** [apache/accumulo#803]
I was trying to convert [ContinuousVerify](https://github.com/apache/accumulo-testing/blob/master/src/main/java/org/apache/accumulo/testing/continuous/ContinuousVerify.java) to use the new builder.  In some cases, `scanOffline` should be set for AccumuloInputFormat but all of the time.  The builder could be created twice in an `if-else` or it could be saved off into variable mid build but what would be better is if you could pass in a boolean to the `scanOffline()` method and have just one use of the builder.  

All methods like this in the builder should be converted to accept a boolean as I think this a common issue.

**Comment by milleruntime** [comment]
Yeah this would be good.  I still think it is pretty clear to do this:

```java
AccumuloInputFormat.configure().clientProperties(props).table(name)
      .scanIsolation(true) // not available with batchScan()
      .offlineScan(booleanVar) // not available with batchScan()
      .store(job);
```

**Comment by milleruntime** [comment]
@mikewalch are you working on this?  If not, I will go ahead and knock it out since it would make #749 a bit easier.

**Comment by mikewalch** [comment]
No I am not
